app/tests/spec/views/mixins/avatar-mixin.js

```diff
@@ -119,11 +119,13 @@ define([
 
     describe('updateProfileImage', function () {
       it('stores the url', function () {
-        view.updateProfileImage(new ProfileImage({ url: 'url' }));
-        assert.equal(account.get('profileImageUrl'), 'url');
-        assert.isTrue(view.getSignedInAccount.called);
-        assert.isTrue(user.setAccount.calledWith(account));
-        assert.isTrue(notifications.profileChanged.calledWith({ uid: UID }));
+        view.updateProfileImage(new ProfileImage({ url: 'url' }))
```

Comment:
Since `view.updateProfileImage` returns a promise, it should be returned to the test here or else the `.then` portion will be run after the test officially completes.